sim/midas/src/main/scala/midas/passes/BridgeTopWiring.scala

```diff
@@ -136,7 +139,8 @@ class BridgeTopWiring(val prefix: String) extends firrtl.Transform {
 
     val topWiringOFileAnno = TopWiringOutputFilesAnnotation("unused", wiringAnnoOutputFunc)
     val topWiringAnnos = topWiringOFileAnno +: inputAnnos.map(_.toWiringAnnotation(prefix)).distinct
-    val wiredState = new TopWiringTransform().execute(state.copy(annotations = topWiringAnnos ++ state.annotations))
+    val wiredState = new Compiler(Forms.MidForm :+ Dependency[TopWiringTransform], Forms.MidForm)
```

Comment:
TopWiring invalidates a bunch of checks and it was easier to specify MidForm again rather than add the checks as targets. Is there an easy easy way to specify "ResolveAndCheck" as a target?